### DISCUSSION TITLE
activation failed no SMS

### DISCUSSION THREAD

**Discussion by ulmpsa208e** [flobz/psa_car_controller#468] [Q&A]
In the Home Assist environment the activation works fine (SMS received ...) and I can see the data of the car. If I follow the description of the stand alone installation it fails. (same user data used) I do not receive a SMS with activation code ... Has there anything changed in the meantime ? Thanks

**Comment by matilbianco** [comment]
Similar issue here but with HA plug-in : I do not receive any SMS although says "SMS sent"
